src/test/scala/mesosphere/marathon/integration/MesosAppIntegrationTest.scala

```diff
@@ -372,5 +378,52 @@ class MesosAppIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonT
       status2 should be(OK)
       status2.value.instances.filter(_.status == raml.PodInstanceState.Stable) should have size 2
     }
+
+    "deploy a simple pod with unique constraint and then " taggedAs WhenEnvSet(envVar, default = "true") in {
```

Comment:
Why is this tagged?

Reply:
It's a Mesos App test, we don't run them on mac (https://github.com/mesosphere/marathon/#testing)

Reply:
@meln1k huh, that's strange. The Mesos (not docker) tests run just fine on a mac for me. The majority of our integration tests run against a Mesos cluster.